src/types.ts

```diff
@@ -39,20 +60,42 @@ export interface ISoundBody {
   backgroundTrackId: string;
 }
 
+export type SectionConfig = {
+  /** Voice name. See the list of available voices using Voice resource. Default voice is "Joanna".
+   */
+  voice?: string;
+  /** Voice speed. Default speed is 100. */
+  speed?: string | number;
+  /** Put a funny effect in your voice. You can try the following ones: dark_father, chewie, 88b, 2r2d */
+  effect?: EffectOptions;
+  /** Add a silence padding to your speech tracks (in milliseconds). Default is 0 (no padding) */
+  silence_padding?: string | number;
+};
+export type PersonalisationParameters = Record<string, string>;
+export type Audience = [PersonalisationParameters];
+export type EffectOptions = "dark_father" | "chewie" | "88b" | "2r2d";
 export interface IMasteringBody {
   scriptId: string;
   backgroundTrackId: string;
-  audience?: [{ [key: string]: string }];
+  audience?: Audience;
 }
 
 export interface IVoiceFilteringBody {
+  /** Try with one of: amazon polly, google, microsoft azure, aflorithmic labs */
   providerFullName?: string;
+  /** Try one of: google, polly, azure, msnr */
   provider?: string;
+  /** Try with one of: male, female */
   gender?: string;
+  /** Try with one of: english, spanish, french, german */
   language?: string;
+  /** Try with one of: american, british, neutral, portuguese/brazilian, american soft, mexican, australian */
   accent?: string;
+  /** Try with one of: adult, child, senior */
   ageBracket?: string;
+  /** Try with one or more (separated by commas) of: steady, confident, balanced, informative, serious, instructional, slow, storytelling, calm, clear, deep, formal, sad, thin, fast, upbeat, fun, energetic, tense, very fast, flat, low pitched, high pitched, low-pitched, sing-y, cooperative, kind, stable, monotonous, neutral, responsible, business man, straight to the point, knowledgeable, focused, newscastery, newsreader, interviewer, reliable, friendly, welcoming, good for handing out information, slightly friendly */
```

Comment:
Lots of tags have been taken out. Can you replace them with:
steady, confident, balanced, informative, serious, storytelling, calm, deep, formal, sad, thin, energetic, upbeat, fun, tense, flat, low pitched, high pitched, fast, slow